i18n/currency-info.php

```diff
@@ -930,7 +930,7 @@
 		'fr_NC'   => $global_formats['rs_comma_space_ltr'],
 		'fr_PF'   => $global_formats['rs_comma_space_ltr'],
 		'fr_WF'   => $global_formats['rs_comma_space_ltr'],
-		'default' => $global_formats['rs_comma_space_ltr'],
+		'default' => $global_formats['ls_dot_comma_ltr'],
```

Comment:
This change should be consistent with https://github.com/woocommerce/woocommerce/pull/46960.